### DISCUSSION TITLE
⚠️ Releases in progress (Robot Testing Framework, YARP and iCub)

### DISCUSSION THREAD

**Discussion by drdanz** [robotology/community#350] [Announcements]
We are about to release Robot Testing Framework, YARP and iCub, please do not update your robot until new instructions.

**Comment by yeshasvitirupachuri** [comment]
Is it safe now? Given https://github.com/robotology/QA/issues/352 @traversaro

**Comment by Nicogene** [comment]
Still don't, we released `master` (`3.1.2`) we have still to release `devel` (`3.2.0`).

**Comment by drdanz** [comment]
Confirming that it is now safe to upgrade.
Sorry for the delay
